templates/download/mediatek.html

```diff
@@ -0,0 +1,109 @@
+{% extends "download/_base_download.html" %}
+
+{% block title %}Install Ubuntu on MediaTek{% endblock %}
+
+{% block meta_description %}Use Ubuntu on MediaTek for the familiar developer experience and an accelerated path to production {% endblock meta_description %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1RMvAI7Soa_zIlP0e63BMcFT3DJfvLrMS2EyiOvcy-xc/edit#{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--suru-topped is-bordered">
+  <div class="row">
+    <div class="col-7">
+      <h1>Install Ubuntu on MediaTek</h1>
+      <p>Run Ubuntu on your MediaTek Genio 1200. </p>
+      <p>Pick the OS image, flash it onto internal EMMC, and away you go.</p>
+      <p><a href="/download/contact-us" class="js-invoke-modal p-button--positive">Get in touch</a></p>
+    </div>
+    <div class="col-5 u-vertically-center u-align--center u-hide--small u-hide--medium">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/0ebb345c-MediaTek_logo.svg",
+        alt="",
+        width="1024",
+        height="268",
+        hi_def=True,
+        loading="auto"
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-7">
+      <h2>Download Ubuntu Server</h2>
+      <h3 class="p-heading--4">Ubuntu Server 22.04 LTS</h4>
+      <p>The version of Ubuntu with up to 10 years of long term support, until April 2032.</p>
+      <p><a href="#" class="p-button--positive">Download 64-bit</a></p>
```

Comment:
The link is not working...

```suggestion
      <p><a href="https://people.canonical.com/~platform/images/xilinx/kria-ubuntu-22.04/iot-limerick-kria-classic-desktop-2204-x06-20220614-78.img.xz" class="p-button--positive">Download 64-bit</a></p>
```